packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -87,6 +90,14 @@ export class VaButton {
   })
   componentLibraryAnalytics: EventEmitter;
 
+  componentDidLoad() {
+    // check if the element has a class named uswds-false added from parent
+    if (this.el.classList.contains('uswds-false')) {
```

Comment:
@it-harrison Do you mind adding some details here as a PR comment about why you're needing to do this?

Reply:
@jamigibbs sure - there is a bug with Stencil in which an attribute set on a child component inside a parent does not get added to the rendered HTML. For instance, if we want the v1 version of `va-button-pair` and we set `uswds="false"` on `VaButtonPair` then, although the `uswds` prop is added to the child `va-button` components inside `va-button-pair`, when rendered the `va-button` elements will not have `uswds="false"`. So it is necessary to add this attribute manually. When all v1 components have been decommissioned we can remove this code where it occurs.